enhancements/helm3/console.md

```diff
@@ -102,10 +101,91 @@ spec:
   disabled: true
 ```
 
-The console operator would watch for changes on it and reconfigure the chart repository proxy. The console backend already implements a few helm endpoints (including the chart proxy), but the future might require to extract them into a separate service (e.g. to make openshift more modular). Hence, it would be good to define the configuration as the top-level resource detaching API from the implementation. Cluster admin would easily discover the new resource and manage it either via CLI or through UI. 
+An operator would watch for changes on them and reconfigure the chart repository proxy. 
+
+---
+**NOTE**
+
+The console backend already implements a few helm endpoints (including the chart proxy), but the future might require to extract them into a separate service (e.g. to make openshift more modular). 
```

Comment:
"..but the future might require to extract them into a separate service (e.g. to make openshift more modular)." to
".. in future, we could extract them into a separate service to make the functionality more modular.."